### ISSUE TITLE
Archives of previous builds making the repository heavy

### ISSUE THREAD

**Issue by lazyvab** [requestly/chrome-extension#136]
@sachinjain024 
Since currently we are archiving all historical builds in **build** folder, cloning or downloading a released version consumes a lot of time and network bandwidth.

1. We should add **build** folder in .gitignore file.
2. As per [0], we can drag the binary while creating release from github. There are also some grunt plugins available to do the same.[1]

[0] - https://help.github.com/articles/creating-releases/ (point#7)
[1] - https://www.npmjs.com/package/grunt-github-releaser

**Comment by sachinjain024** [comment]
Thanks @lazyvab for logging this. I have removed earlier builds from repo with this checkin (https://github.com/requestly/chrome-extension/commit/abe0b55aa46c0533de4a94b8155b1e941ffd3cd5).
I have also added a tag so that in case we need earlier builds, we can directly checkout that tag and go one commit earlier. (https://github.com/requestly/chrome-extension/releases/tag/Issue-136-delete-older-builds).

Need to update Requestly release process so that we keep on deleting earlier builds when we draft a new build.
